apps/omg_child_chain/lib/omg_child_chain/block_queue/core.ex

```diff
@@ -358,20 +358,22 @@ defmodule OMG.ChildChain.BlockQueue.Core do
          %Core{
            parent_height: parent_height,
            last_enqueued_block_at_height: last_enqueued_block_at_height,
-           minimal_enqueue_block_gap: minimal_enqueue_block_gap,
+           block_submit_every_nth: block_submit_every_nth,
            wait_for_enqueue: wait_for_enqueue
          },
          is_empty_block
        ) do
-    it_is_time = parent_height - last_enqueued_block_at_height > minimal_enqueue_block_gap
+    # e.g. if we're at 15th Ethereum block now, last enqueued was at 14th, we're submitting a child chain block on every
+    # single Ethereum block (`block_submit_every_nth` == 1), then we should mine a block (`it_is_time` is `true`)
```

Comment:
To clarify the message, how about using just `enqueue` or just `mine` verb to describe action performed on blocks #14 and #15?

Reply:
these mean a different thing. But, current wording is indeed wrong, should be `then we should form a block`, mining is confusing.

But then `enqueue` vs `form` is significant. We `form` if a certain amount of eth blocks are `mined` after our last child block `enqueued`.